### PR TITLE
Issue#231  align selects on list of meeting members

### LINKED ISSUE #231: List of meeting members: align Selects
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**

Regarding the management of members of a meeting, the owner of a meeting cannot be removed from the meeting. And so it is not necessary to add the menu.
But, the fact of not adding the menu leads to a non-alignment of the Select which defines the role of the owner of the meeting with the Select of the other members.

**Describe the solution you'd like**
For users who do not have menus, in this case the owner of the meeting, instead of the menu, it would be necessary to have an empty Box which serves as a filling in order to balance the offset.

**Describe alternatives you've considered**
N/A

**Discovery, Documentation, Adoption, Migration Strategy**
N/A

**Do you want to work on it through a Pull Request?**
yes!

### PR DESCRIPTION
**Pull request by adriennickson**
Resolve #231, close #231 

## Purpose
Related to the issue #231

## Proposal
The proposed solution is to replace the missing menu with an empty Box component.

### CODE REVIEW

File: src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx [hunk 1]
```diff
@@ -14,6 +14,14 @@ describe('RowUserBase', () => {
     await screen.findByText(randomUser.name);
   });
 
+  it('basic render with empty Box instead of Menu if there is not actions defined', async () => {
+    const randomUser = createRandomUser();
+    renderWrappedInTestingProvider(<UserRowBase user={randomUser} />);
+    const menuBox = await screen.findByTestId('menuBox')
+    const emptyBox = menuBox.querySelector(".emptyBox")
+    expect(emptyBox).toBeInstanceOf(HTMLElement);
```
Comment (PanchoutNathan) [inline src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx:22]: Why not directly search for the presence of the element by also adding a data-testid to it?

```
await screen.findByTestId('emptyBox')
```

File: src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx [hunk 2]
```diff
@@ -17,8 +17,8 @@ describe('RowUserBase', () => {
   it('basic render with empty Box instead of Menu if there is not actions defined', async () => {
```
Comment (PanchoutNathan) [inline src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx:17]: Squash this commit with the previous commit.

File: src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx (hunk 1 above)
Comment (adriennickson) [reply src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx:21]: Hi @PanchoutNathan thanks for feedback 🙏🏾.
I just pushed updates.

File: src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx (hunk 2 above)
Comment (adriennickson) [reply src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx:17]: Done

File: Makefile [hunk 3]
```diff
@@ -197,6 +202,7 @@ superuser: ## Create an admin user with password "admin"
 	@$(COMPOSE) up -d mysql
 	@echo "Wait for services to be up..."
 	@$(WAIT_DB)
+	@$(MANAGE) migrate
```
Comment (sampaccoud) [inline Makefile:205]: I don't think we want to force migrate when creating a user. Can you remove this change?

Comment (adriennickson) [reply Makefile]: Yes, sure.
Done

File: setup.cfg [hunk 4]
```diff
@@ -58,12 +58,12 @@ dev =
     ipdb==0.13.13
```
Comment (PanchoutNathan) [inline setup.cfg:58]: Put this in a separate commit with the title: ``⬆️(dependencies) update python dependencies``

File: Makefile [hunk 5]
```diff
@@ -120,6 +120,11 @@ build-front:
 	@$(COMPOSE_RUN) -e HOME="/tmp" -w /app/src/frontend node yarn install
 	@$(COMPOSE_RUN) -e HOME="/tmp" -w /app/src/frontend node yarn build
 
+run-front: ## start frontend development server with live reload
```
Comment (PanchoutNathan) [inline Makefile:123]: Put this in separate commit with this `🔧`emoji

File: src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx (hunk 1 above)
Comment (PanchoutNathan) [reply src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx:21]: you can remove ```expect(emptyBox).toBeInstanceOf(HTMLElement);```
Because the return type of  
```const emptyBox = await screen.findByTestId('emptyBox');```   
is a ```Promise<HTMLElement>``` and fails the test if it doesn't find it.

File: setup.cfg (hunk 4 above)
Comment (adriennickson) [reply setup.cfg:58]: My bad ^^. I squashed my commits with the boot one.

Done

**Review by PanchoutNathan — APPROVED** [review]
Resolve all conversations and LGTM (Let's go to the moon) !
